cypress/support/pages/user.pageObject.js

```diff
@@ -0,0 +1,25 @@
+import PageObject from '../PageObject';
+
+class userPageObject extends PageObject {
+  visitUserPage(username) {
```

Comment:
It is recommended to use single quotes instead of backticks when you are not using interpolation.